src/backend/app.js

```diff
@@ -21,6 +21,61 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//after the line where it says router.use("/meals", mealsRouter);. There you can go ahead and define the desired routes like you normally would:
+
+app.get("/my-route", (req, res) => { res.send("Hi friend") });
+
+const knex = require('knex')({
+  client: 'mysql2',
+  connection: {
+      host : '127.0.0.1',
+      port: 3306,
+      user: 'root',
+      password: '12345678',
+      database: 'meal_sharing'
+  }
+})
+//Respond with all meals in the future (relative to the when datetime)
+app.get('/future-meals', async(req, res) => {
+  const dbResult = await knex.raw('SELECT * FROM meal WHERE when_date > NOW()')
+  const rows = dbResult[0]
+  res.json({
```

Comment:
I am not sure if these braces around roiws are required. 
res.json(rows) should works just fine

Reply:
Yes Its not required  as I have updated in my last commit But thank for noticing 😊